migration/20170224-2-copy-collection-configuration-into-database.py

```diff
@@ -114,8 +114,8 @@ def convert_rbdigital(_db, library):
 
     collection, ignore = get_one_or_create(
         _db, Collection,
-        protocol=Collection.RBDIGITAL,
-        name="RBDigital"
+        protocol=Collection.ONECLICK,
+        name="OneClick"
```

Comment:
It turns out this whole thing is broken, because we changed the way Collection and ExternalIntegration relate to each other. Since everyone who was going to run this code has already run it, it's fine to leave the whole file as is.